sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsView.java

```diff
@@ -92,6 +92,9 @@
 	private Label relevanceStatusInfoLabel;
 	private ComboBox relevanceStatusFilter;
 
+	// calculation method to use for contact counts
```

Comment:
I don't think the comment is necessary here, the field is pretty self-explanatory; if you want to use a comment, please use a JavaDoc style comment instead (since it's about describing a class field)